### PR TITLE
Changed 2x stake filter text

### LINKED ISSUE #615: "2x Stake" is confusing; rename it
## Problem
The following title is misleading "2x Stake" is misleading.
It makes me think as a user that I'll receive a bonus if I lock up to these datasets.

## Solution
Rename it to something more appropriate: "Boosted".
Extracted from the language of: "effective 2x allocation boost"

image

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #615.

### CODE REVIEW

**Review by idiom-bytes — APPROVED** [review]
LGTM

File: src/utils/data.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export const filterOptions = [
     { id: "0", text: "All datasets" },
     { id: "1", text: "My allocations" },
-    { id: "2", text: "2X Stake" },
+    { id: "2", text: "Publisher 2X stake" },
```
[the remark this answers is not in the document]
Comment (KatunaNorbert) [reply src/utils/data.js:4]: In terms of length it sounds good, but it's quite ambiguous.
For example if you are on the DCV and have published 2x selected what does that mean, is it 2x DCV?

[the remark this answers is not in the document]
Comment (idiom-bytes) [reply src/utils/data.js:4]: Publisher 2x specifically means that you get your veOCEAN effectively doubled during the reward calculations every week.

It's [well documented in the docs](https://docs.oceanprotocol.com/~/revisions/E1xWAzXFffFRpE9EP7w4/rewards/df-volumedf#what-are-publisher-rewards) and in blog posts


When you select this, users are able to see that these assets normally have high veOCEAN allocated to them.
image

[To improve visualizing this bonus]
1. If the user clicked on the row or hovered, it could be shown that:
- Total Allocation (veOCEAN)
- Publisher Allocation (veOCEAN)
- Publisher Bonus (veOCEAN *2)

2. Publisher Bonuses could also be visualized via a DF round summary chart.

TLDR; Improving this visualization at the moment is not important.

**Review by idiom-bytes — APPROVED** [review]
LGTM
